### ISSUE TITLE
Clarification on Visible Controls (AA) - (Visible vs Perceivable)

### ISSUE THREAD

**Issue by lloydi** [w3c/wcag#2321]
For 3.2.7, the description is "Where receiving pointer hover or keyboard focus triggers user interface components to be visible, information needed to identify that user interface components are available is visible,"

In a follow-up to an example of where 1.4.1 Use of Color is *not* a failure in this article https://www.tpgi.com/well-color-us-surprised-this-sc-can-be-a-tricky-customer/ (see Test 6 where link color is same as adjacent paragraph text) I got this tweet:

https://twitter.com/alvaro_montoro/status/1514221215740792839?s=20&t=7UG8X029w8ja8yLV1vRbMQ

In the following tweets, Sara Horton suggested that 3.2.7 might answer this case but I am not sure.

In my example, the links are visible but not clearly identifiable as something that can be interacted with (until you hover over them or focus on them). I think this is NOT what is being suggested by 3.2.7. The links in my example *are* visible, just not perceivable as links. 

Is my interpretation of this correct? That 3.2.7 is not (or will not be) applicable here?

**Comment by detlevhfischer** [comment]
In my view, it would fall under 3.2.7 Visible Controls, so I agree with Sara  - while the link text is always visible, it is not discernable from surrounding text, i.e. it's _character of being a control_ is not apparent without focus.

**Comment by alastc** [comment]
"When user interface components are **invisible** until hover" 

It is not a case where hovering causes controls to appear, so it wouldn't be covered by 3.2.7.

**Comment by lloydi** [comment]
OK, thanks Alistair. Will watch for any further feedback :)

**Comment by alastc** [comment]
Visible controls has been deferred (implemented in #2353), so closing.
